Eclecticiq-ic-app/metadata/local.meta

```diff
@@ -0,0 +1,59 @@
+[views/home]
```

Comment:
There should be no local.meta. Remove it.

Reply:
Okay